Makefile

```diff
@@ -196,9 +196,9 @@ ifneq ($(SKIP_BOOTSTRAP),1)
 # If we're in a git worktree, the git hooks directory may not be in our root,
 # so we ask git for the location.
 #
-# Note that this command requires git 2.5+; earlier versions produce opaque
-# errors.
-GITHOOKSDIR := $(shell git rev-parse --git-path hooks)
+# Note that `git rev-parse --git-path hooks` requires git 2.5+, so we catch the error
+# produced if git is an earlier version and default to `.git/hooks`.
+GITHOOKSDIR := $(shell (git rev-parse --git-path hooks &>/dev/null && git rev-parse --git-path hooks) || echo '.git/hooks')
```

Comment:
Maybe I'm just getting confused by the desired behavior of that command, but was the goal for it to fail if the git hooks directory is not in our root and Git < 2.5? If so then I'll make the change.